### DISCUSSION TITLE
Have you considered open sourcing your documentation generator?

### DISCUSSION THREAD

**Discussion by will-stone** [adobe/react-spectrum#7007] [Ideas]
The docs for RA are awesome! I've not yet found a tool that shows such rich integration into TypeScript (Storybook doesn't display deep type info). I can see you use the `@react-spectrum/docs` package, which looks to be private, any designs on releasing it?

**Comment by snowystinger** [comment]
We've talked about it here and there, but it isn't high on our priorities, unfortunately. It'd probably have to be a community contribution starting with a RFC https://react-spectrum.adobe.com/contribute.html#feature-requests as we'd probably need to generalize a bunch of things and figure out APIs.
